RedHat-UX/red-hat-design-system
# chore: don't error when bundling empty dirs

## What I did

1. prevent `scripts/bundle.js` from erroring when it encounters empty directories. This will fix the compressed size action on PRs with new files.


## Testing Instructions

1. add an empty dir `elements/rh-jazz-hands` then `npm run build` - bundle script should not error.
1. merge this then update the audio-player PR. compressed size action should not fail

## Notes to Reviewers
